src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -138,11 +143,14 @@ class CollapsibleCourseRun extends React.Component {
       element.scrollIntoView({ behavior: 'smooth', block: 'start', inline: 'nearest' });
     }
 
+    if (currentType !== prevProps.type) {
+      change(`${courseId}.run_type`, courseRunTypeOptions[currentType][0].value);
+    }
```

Comment:
I'm not familiar with what this will do in redux land.

Reply:
This is just going to dispatch the `CHANGE` action to update the current form value rather than me hard code it in as `currentFormValues.run_type = _`. I believe it is a cleaner way of doing it